RFS/src/test/java/org/opensearch/migrations/bulkload/common/OpenSearchClientFactoryTest.java

```diff
@@ -58,6 +58,7 @@ class OpenSearchClientFactoryTest {
     private static final String CLUSTER_SETTINGS_COMPATIBILITY_OVERRIDE_ENABLED = "{\"persistent\":{\"compatibility\":{\"override_main_response_version\":\"true\"}}}";
     private static final String CLUSTER_SETTINGS_COMPATIBILITY_OVERRIDE_DISABLED = "{\"persistent\":{\"compatibility\":{\"override_main_response_version\":\"false\"}}}";
     private static final String ROOT_RESPONSE_OS_1_0_0 = "{\"version\":{\"distribution\":\"opensearch\",\"number\":\"1.0.0\"}}";
+    private static final String ROOT_RESPONSE_OS_3_0_0_alpha = "{\"version\":{\"distribution\":\"opensearch\",\"number\":\"3.0.0-alpha1\"}}";
```

Comment:
Should we still be using the alpha string here?

Reply:
Ah, I see that this is basically just to test getting a version input with `-alpha`